src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/PXGraph/DataViewInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Acuminator.Utilities.Roslyn.Semantic.PXGraph
+{
+    public readonly struct DataViewInfo
+    {
+        public ISymbol Symbol { get; }
```

Comment:
It would be nice to add some XML comments here because it is not obvious what `Symbol` and `Type` mean

Reply:
Ok